linux-tools/libndp/libndp.py

```diff
@@ -0,0 +1,168 @@
+#!/bin/python
+import aexpect
+from distutils.spawn import find_executable
+from autotest.client import test, utils
+from autotest.client.shared import error
+
+
+class libndp(test.test):
+
+    """
+    libndp -Library for Neighbor Discovery Protocol
+
+    Neighbor Discovery Protocol (NDP) is a part of the Internet Protocol suite used with IPv6.
+    NDP Protocol make use of ICMPv6 messages and solicited-node multicast addresses for operating    its core function, which is tracking and discovering other IPv6 hosts that are present on the    other side of connected interfaces.
+
+    libndp package contains a library,libndp.so :which provides a wrapper for IPv6 Neighbor         Discovery Protocol and ndptool :tool for sending and receiving NDP messages.
+
+    @author Ramya BS <ramya@linux.vnet.ibm.com>
+
+    """
+    version = 2
+    nfail = 0
+    interface = utils.subprocess.Popen(
```

Comment:
`utils.subprocess` is an internal import that should not be imported. Just because it's available on that namespace, doesn't mean it should be used here. You should be doing simply import subprocess.